operate.Dockerfile

```diff
@@ -1,7 +1,7 @@
 # hadolint global ignore=DL3006
 ARG BASE_IMAGE="alpine:3.19.1"
-ARG BASE_DIGEST_AMD64="sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b"
-ARG BASE_DIGEST_ARM64="sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b"
```

Comment:
@houssain-barouni where do you get these shas from? 🤔 

Because sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b I can see in dockerhub https://hub.docker.com/layers/library/alpine/3.19.1/images/sha256-a0264d60f80df12bc1e6dd98bae6c43debe6667c0ba482711f0d806493467a46?context=explore

Reply:
Ah I see the manifest digest sha256:6457d53fb065d6f250e1504b9bc42d5b6c65941d57532c072d929dd0628977d0